perun-core/src/main/resources/hsqldbChangelog.txt

```diff
@@ -8,6 +8,10 @@
 
 -- this update is not supported on hsql since its used only as in-memory db
 
+3.1.54
+update configurations set value='3.1.54' where property='DATABASE VERSION';
```

Comment:
Version update in sql changelog should be last command (all three files).

Reply:
Order was fixed.